modules/tsb/cp/manifests/tsb/controlplane-values.yaml.tmpl

```diff
@@ -78,7 +87,13 @@ spec:
         redis:
           uri: redis-master.${ratelimit_namespace}.svc.cluster.local:6379
     %{ endif }
-  meshExpansion: {}
+  meshExpansion:
+    onboarding:
+      localRepository: {}
+      endpoint:
+        hosts:
+          - ${vm_endpoint}
+        secretName: vm-onboarding
```

Comment:
Opened as draft to see if it is OK to configure this via ext-dns. If the approach is OK I'll add the bots to generate the certificate and populate this secret, which is still not in this PR.

Reply:
oh yeah, so heavily looking forward to the trusted certificate :)